graphicalquerybuilder/dreyfus_wagner.hpp

```diff
@@ -0,0 +1,318 @@
+//=======================================================================
```

Comment:
@Maxzor is this file really needed here? It seems a boost module or something like this...

Reply:
It is a paal file, I modified it, I made public the entire class because I wanted to access most properties from graphicalquerybuilderpath.cpp
The setup.sh script takes it and move it into paal to replace the existing.
If you have ideas I am all ears!